editor/editor_node.cpp

```diff
@@ -31,6 +31,7 @@
 #include "editor_node.h"
 
 #include "core/config/project_settings.h"
+#include "core/core_string_names.h"
```

Comment:
```suggestion
```
Already included

Reply:
Again, I'm not sure what you mean by this. Removing this `#include` makes it impossible to access CoreStringNames in this context as far as I can tell.